### ISSUE TITLE
[CT-317] [Feature] <Link description of models to custom overview>

### ISSUE THREAD

**Issue by jan-benisek** [dbt-labs/dbt-core#4818]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

I am adding [custom overview](https://docs.getdbt.com/docs/building-a-dbt-project/documentation#setting-a-custom-overview) to our project, which works great.

For some longer columns in our models, we are using `.md` files with `{% docs foo_bar %}`, just like the [dbt documentation](https://docs.getdbt.com/docs/building-a-dbt-project/documentation#usage) shows.

I'd like to be able to link the content of the above-mentioned file to the custom overview.
_(example is a list of countries and our particular grouping - columns in a model. This is important to be easily visible)_

I'd also like to show `selectors.yaml` in the custom overview, to display which models are currently being run.

Basically, I'd like to keep things DRY and want the `.md` overview file render jinja commands, like `{% docs %}` and `.yml`.

### Describe alternatives you've considered

AFAIK no such feature exists, so I am simply copy/pasting the content. Obviously this is not great.

### Who will this benefit?

In our case, our less tech savvy colleagues who consult `dbt docs`. It helps them get relevant information quickly.
On the other hand, as a developer, this would keep the docs DRY and easy to maintain should anything change.

### Are you interested in contributing this feature?

if it is easy(-er) I could give it a shot

**Comment by emmyoop** [comment]
Thanks for writing this up @jan-benisek.  I'm glad you've been able to take advantage of the custom overview and longer `docs` blocks.

You can link to specific models in the overview since all resources in the docs site have URLs like:

```[base url]/#!/[resource_type]/[resource_type].[package_name].[resource_name]```

So while you cannot link to a specific docs block, you could link to the model that contains that block with something like: 

```
You can see this awesome explanation of groupings in the description of [countries_model](/#!/model/model.my_project.countries_model) and this [macro](/#!/macro/macro.my_project.awesome_macro).
```

I know this is not exactly what you're asking but hopefully it could bridge the gap.  

As things stand today, we don't have access to the `selectors.yml` so copying it over manually or via some script you maintain is how it will end up in your overview file.
